### PR TITLE
New kickoff

### PR DESCRIPTION
**Pull request by mpicollo-wolox**
## Summary

- Add yeoman generator for new version of bootstrap, includes: 

    - [x] Last changes in master
    - [x] [New linter](https://github.com/Wolox/express-js-bootstrap/pull/96)
    - [x] [Jest or chai](https://github.com/Wolox/express-js-bootstrap/pull/92)
    - [x] [New migration script](https://github.com/Wolox/express-js-bootstrap/pull/88)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require('eslint-config-wolox-node');
```
Comment (gonziesc) [inline .eslintrc.js]: love u

**Comment by gonziesc** [comment]
for me this is ok, talks with @fede1608 and the rest of the tema to pass this

File: generators/app/templates/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Wolox
```
Comment (arinaldi118) [inline generators/app/templates/LICENSE.md]: 2019 :eyes: :eyes:

Comment (mpicollo-wolox) [reply generators/app/templates/LICENSE.md]: 👮

**Pull request closed without merging**